engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/behavior/BpmnStateTransitionBehavior.java

```diff
@@ -371,14 +379,16 @@ public void onCalledProcessTerminated(
   }
 
   public void afterExecutionPathCompleted(
-      final ExecutableFlowElement element, final BpmnElementContext childContext) {
+      final ExecutableFlowElement element,
+      final BpmnElementContext childContext,
+      final boolean satisfiesCompletionCondition) {
```

Comment:
❌  this is breaking some tests:

> Cannot invoke "java.lang.Boolean.booleanValue()" because the return value of "io.camunda.zeebe.util.Either.get()" is null

```suggestion
      final Boolean satisfiesCompletionCondition) {
```

Please make sure to change it also in `BpmnElementContainerProcessor`

Reply:
Oops! my bad.